doc/nrf/ug_thread_architectures.rst

```diff
@@ -0,0 +1,160 @@
+
+   Thread-only architecture
+
+.. _thread_architectures_designs_soc_designs_multiprotocol:
+
+Single-chip, multiprotocol (SoC)
+================================
+
+With nRF52840 and nRF52833 supporting multiple wireless technologies, including IEEE 802.15.4 and Bluetooth Low Energy (|BLE|), the application layer and OpenThread still run on the same processor.
+
+In this multiprotocol design, the SoC ensures either dynamic or switched Thread and |BLE| connectivity.
+
+This design has the following advantages:
+
+* It leverages the benefits of highly integrated SoC, resulting in the lowest cost and the lowest power consumption.
+* It allows to run Thread and |BLE| simultaneously on a single chip, which reduces the overall BOM cost.
+
+It also has the following disadvantages:
+
+* |BLE| activity can degrade the connectivity on Thread if not implemented with efficiency in mind.
+
+.. figure:: /images/thread_platform_design_multi.svg
+   :alt: Multiprotocol Thread and |BLE| architecture
+
+   Multiprotocol Thread and |BLE| architecture
+
+.. _thread_architectures_designs_cp:
+
+Co-processor designs
+********************
+
+In the co-processor designs, with either network co-processor (NCP) or radio co-processor (RCP), the application layer runs on a host processor and communicates with OpenThread through a serial connection using a standardized host-controller protocol (Spinel).
+OpenThread can run on either the radio or the host processor.
+
+.. _thread_architectures_designs_cp_ncp:
+
+Network Co-Processor (NCP)
+==========================
+
+The standard NCP design has Thread features on the SoC and runs the application layer on a host processor, which is typically more capable than the OpenThread device, although it has greater power demands.
+The host processor communicates with the OpenThread device through a serial interface (typically UART or SPI) over the Spinel protocol.
+
+This design is useful for gateway devices or devices that have other processing demands, like IP cameras and speakers.
+
+This design has the following advantages:
+
+* The higher-power host can sleep, while the lower-power OpenThread device remains active to maintain its place in the Thread network.
+* Since the SoC is not tied to the application layer, development and testing of applications is independent of the OpenThread build.
+* Only the network stack and a thin application reside on the NCP, which reduces the cost of the chip (RAM and Flash usage may be smaller than in an SoC solution with the application).
+* It does not require the support for the dual-bank DFU.
+  (Host can just replace the old image with a new one.)
+
+It also has the following disadvantages:
+
+* This is the most expensive option, since it requires the application processor.
+
+.. figure:: /images/thread_platform_design_ncp.svg
+   :alt: Network Co-Processor architecture
+
+   Network Co-Processor architecture
+
+.. note::
+    Spinel connection through USB is not currently available.
```

Comment:
Need to verify that but I think SPI is not available as well

Reply:
That's true. Currently we have only UART transport in the NCS for NCP.